web/src/components/Btn.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Btn = ({ extraCss, kind, size, ...rest }) => {
+  const classes = ['btn', `btn-${kind}`, size && `btn-${size}`, extraCss]
+    .filter(c => c)
+    .join(' ');
```

Comment:
Ha, nice.  I guess this prevents us from having extra spaces in the class property?